pkg/console/subresource/deployment/deployment.go

```diff
@@ -332,3 +309,46 @@ func IsAvailableAndUpdated(deployment *appsv1.Deployment) bool {
 		deployment.Status.ObservedGeneration >= deployment.Generation &&
 		deployment.Status.UpdatedReplicas == deployment.Status.Replicas
 }
+
+func defaultVolumeConfig() []volumeConfig {
+	return []volumeConfig{
+		{
+			name:     ConsoleServingCertName,
+			readOnly: true,
+			path:     "/var/serving-cert",
+			isSecret: true,
+		},
+		{
+			name:     ConsoleOauthConfigName,
+			readOnly: true,
+			path:     "/var/oauth-config",
+			isSecret: true,
+		},
+		{
+			name:        api.OpenShiftConsoleConfigMapName,
+			readOnly:    true,
+			path:        "/var/console-config",
+			isConfigMap: true,
+		},
+		{
+			name:        api.ServiceCAConfigMapName,
+			readOnly:    true,
+			path:        "/var/service-ca",
+			isConfigMap: true,
+		},
+	}
+}
+
```

Comment:
`hasCustomLogo` is pretty simplistic compared to `ValidateCustomLogo`, it might be good to update....

Reply:
but is kinda do whats it's called to do :D

Reply:
meaning I would like to use the same validate rules/func, to ensure we don't get a mount when the sync was invalidated. I have not experienced in manual testing, but will tweak this a bit quick.